### PR TITLE
The Condiment Update (feat. Whipped Cream, Butter, Maple Syrup, PANCAKES, and a lot of other stuff actually now)

### LINKED ISSUE #30350: Feature Request: Whipped Cream
Please either add canned whipped cream, or be able to use a spray bottle to apply carbonated whipped cream to food. It's quite sad to eat a slice of pumpkin pie without whipped cream on top. 

It would just be a little white swirl that could be transposed on any food item, but would do wonders for muh immulsion.

### PR DESCRIPTION
**Pull request by DeityLink**
Fixes #30350

sauceimage
image
imageimage
image

:cl:
* rscadd: Cream, Liquid(grinded) Butter, Maple Syrup and Chill Wax are now considered condiments, and can be processed inside their own custom condiment bottles at a CondiMaster
* rscadd: Most condiments now add overlays to the food items they are applied on using condiment bottles. The overlay appears where you click at and you can add as many as you want. (RonPaul08)
* imageadd: Honey, Royal Jelly, and Chill Wax pots have the honey on their sprite change color depending on the reagents mixed in them, which also affects the color of the overlay when they are applied on food items.
* tweak: The whipped cream visible on pumpkin pie and brain cake slices has been removed. You can re-add it manually by adding some whipped cream from a condiment bottle.
* imageadd: Coco, Mustard, Mayo, and Caramel condiment bottles now have in-hand sprites.
* tweak: You can no longer slip condiment bottles inside sliceable cakes
* rscadd: Added Pancakes! Cooked by just pouring 5 units of pancake mix in a microwave. Pancake mix itself is obtained by mixing 10 units of flour, 30 units of milk, 2 units of liquid butter (1/4th of a single grinded butter), 8 units of egg yolk (2 grinded eggs), 1 unit of salf, and 5 units of sugar, resulting in 56 units of pancake mix. That leaves a bit more paste than you can cook with but anyone who's ever cooked pancakes knows you always have a bit of leftover in your bowl. Just mix some more.
* rscadd: Pancakes can be stacked up, up to 10 total. By alternating condiments such as honey or maple syrup and extra pancakes you can produce some delicious looking and extremely fat pancake towers. 
* bugfix: Fixed frost oil being red instead of blue.

### CODE REVIEW

**Comment by jwhitak** [comment]
Liquid butter. Do a liquid butter overlay, Give it the condiment love. for the love of JESUS. I NEED IT. I'M AMERICAN. I NEED TO PUT BUTTER MY EVERYTHING

**Comment by DeityLink** [comment]
sauce

Added liquid butter as a condiment and also you can now click where you want to place the overlay

**Comment by BomberBro** [comment]
mayo

**Comment by Rabbidx7** [comment]
I love it, add syrup. Also add PANCAKES.

**Comment by RonPaul08** [comment]
Based

**Comment by ghost** [comment]
>ketchup on steak

downboated

**Comment by DeityLink** [comment]
image

This PR is going places

**Comment by DeityLink** [comment]
And because why not the color of the honey on pots and overlays changes depending on the other reagents in there as well, just like honeycombs already do. 
image

**Comment by DeityLink** [comment]
And yes you can splice condiments in between pancakes for a nicer look. Also chillwax is now also considered a condiment and uses the same container as honey, since the sprite is now reagent-colored.
image

**Comment by kilozombie** [comment]
Wondrous PR. Just wonderful
Maple syrup is exceptionally annoying and difficult to get at present because it was part of the "food megaproject", the poutine citadel. This could easily be changed, and perhaps should be? Not *fully* atomic, but it'd be cool to be able to put syrup on stuff without it being a sordid affair that involves literally using a snowball.

**Comment by DeityLink** [comment]
but think how more delicious maple syrup would be if it's that rare!

That being said it being now a condiment with its own spawnable item means that you also have a chance to get it from cargo.

**Comment by DeityLink** [comment]
don't merge yet, got more to add, but first, sleep

**Comment by DeityLink** [comment]
mayo inhand 
image

**Comment by DeityLink** [comment]
image

**Comment by DeityLink** [comment]
Alright I'm done now.

File: code/modules/food/recipes_microwave.dm [hunk 1]
```diff
@@ -386,6 +386,18 @@
 		)
 	result = /obj/item/weapon/reagent_containers/food/snacks/rofflewaffles
 
+/datum/recipe/rofflewaffles
+	reagents = list(PSILOCYBIN = 5, FLOUR = 10)
+	items = list(
+		/obj/item/weapon/reagent_containers/food/snacks/egg,
+		/obj/item/weapon/reagent_containers/food/snacks/egg,
+		)
+	result = /obj/item/weapon/reagent_containers/food/snacks/rofflewaffles
```
Comment (Eneocho) [inline code/modules/food/recipes_microwave.dm:395]: @DeityLink It seems you duplicated the rofflewaffles recipe for some reason.

**Comment by Eneocho** [comment]
@DeityLink, what if you use the Vox Condiment dispenser? Does the food get an appropriate overlay?

File: code/modules/food/recipes_microwave.dm (hunk 1 above)
Comment (DeityLink) [reply code/modules/food/recipes_microwave.dm:395]: woops

**Comment by DeityLink** [comment]
> 
> 
> @DeityLink, what if you use the Vox Condiment dispenser? Does the food get an appropriate overlay?

The Vox Condimaster has a unique sprite but works just like any other Condimaster.

**Comment by SonixApache** [comment]
this is actually a really really cool feature and I applaud you for it

**Pull request merged**